lib/libraries/cms/controller/form.php

```diff
@@ -298,7 +298,7 @@ public function cancel($key = null)
 		$app = JFactory::getApplication();
 		$model = $this->getModel();
 		$table = $model->getTable();
-		$checkin = property_exists($table, 'checked_out');
```

Comment:
Yes it does. I'm using the https://github.com/joomlatools/joomlatools-platform/blob/feature/268-concurrency/lib/libraries/joomla/table/table.php#L1261 method.

I'm I missing something here ? This should work no ? Didn't change the logic just moved things around a bit.